### PR TITLE
Openpilot lane line indicators support for Subaru global dash display

### PR DESCRIPTION
**Pull request by martinl**
This PR adds Openpilot lane lines visible indicators support for Subaru global dash display. Tested and working with Subaru XV 2018

### CODE REVIEW

File: selfdrive/car/subaru/subarucan.py [hunk 1]
```diff
@@ -43,12 +43,22 @@ def create_es_distance(packer, es_distance_msg, pcm_cancel_cmd):
...
 
-def create_es_lkas(packer, es_lkas_msg, visual_alert):
+def create_es_lkas(packer, es_lkas_msg, visual_alert, left_line, right_line):
 
   values = copy.copy(es_lkas_msg)
   if visual_alert == VisualAlert.steerRequired:
     values["Keep_Hands_On_Wheel"] = 1
 
+  if left_line:
+    values["LKAS_Left_Line_Visible"] = 1
```
Comment (rbiasini) [inline selfdrive/car/subaru/subarucan.py]: `values["LKAS_Left_Line_Visible"] = int(left_line)`

**Comment by rbiasini** [comment]
Nice. Just one comment I would resolve before merge.
Also, for the dbc changes, can you please open a PR in `opendbc` repo first? That needs to be merged before this can.

File: panda/board/safety/safety_subaru.h [hunk 2]
```diff
@@ -116,13 +116,18 @@ static int subaru_fwd_hook(int bus_num, CAN_FIFOMailBox_TypeDef *to_fwd) {
...
     // global platform
+    // ES LKAS
     if (addr == 0x122) {
       return -1;
     }
     // ES Distance
     if (addr == 545) {
       return -1;
     }
+    // ES LKAS State
```
Comment (rbiasini) [inline panda/board/safety/safety_subaru.h]: Nice catch

**Comment by rbiasini** [comment]
Ended up cleaning this PR and merging it internally. Will be in next OP release. Thanks.

File: selfdrive/car/subaru/subarucan.py (hunk 1 above)
Comment (martinl) [reply selfdrive/car/subaru/subarucan.py]: I tested the suggested lane lines change, it is working. Should I create new PR vs 0.5.12?

**Pull request closed without merging**